bridge/src/org/sparkboard/js_convert.cljs

```diff
@@ -0,0 +1,26 @@
+(ns org.sparkboard.js-convert
+  (:refer-clojure :exclude [clj->js])
```

Comment:
Why the exclusion? I don't see any shadowing.

Reply:
was from before - will remove, thx